RELEASE.md

```diff
@@ -0,0 +1,12 @@
+# Jaeger Backend Release Process
+
+1. Create a PR "Preparing release X.Y.Z" against master branch (e.g. #351). It must include:
```

Comment:
link #351 does not work

Reply:
The text does not mention `changelog.md` file.

Reply:
the link works for me - what do you get?

Reply:
When I hit the view it does now work https://github.com/jaegertracing/jaeger/blob/1a684a8a04e6e8ddd02026760cbe52257a9e6830/RELEASE.md